tests/commands/logs.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+const { spawnSync } = require('child_process');
+
+import {
+  CLASP,
+  SCRIPT_ID,
+} from '../constants';
+
+import {
+  cleanup,
+  setupWithoutGCPProject,
+} from '../functions';
+
+describe('Test clasp logs function', () => {
+  before(setupWithoutGCPProject);
+  it('should prompt for logs setup', () => {
+    const result = spawnSync(
+      CLASP, ['logs'], { encoding: 'utf8' },  // --setup is default behaviour
+    );
+    expect(result.stdout).to.contain('What is your GCP projectId?');
+  });
+  it('should prompt for logs setup', () => {
+    const result = spawnSync(
+      CLASP, ['logs', '--setup'], { encoding: 'utf8' },
+    );
+    expect(result.status).to.equal(0);
+    expect(result.stdout).to.contain('Open this link:');
+    expect(result.stdout).to.include(`https://script.google.com/d/${SCRIPT_ID}/edit`);
+    expect(result.stdout).to.contain('Go to *Resource > Cloud Platform Project...*');
```

Comment:
It seems fragile to have a test depend on all of this specific text. Ideally we'd be able to rephrase these strings without breaking tests. Can we just omit these expects?

Reply:
I agree, the source code should probably use strings from here:

https://github.com/google/clasp/blob/190480c9bfbdad4b944fbb0a9c1c17f273d128b8/src/utils.ts#L124

but as seen in comment above, it just uses those strings as-is. I will fix that.

Reply:
I just don't see why it's necessary to test each of those lines at all. I think the test is trying to confirm that the command will print the setup text, but you know that's true after you check the first line.

Reply:
Great point, I will change it to just expect the prompt "What is your GCP projectId?"